src/snowflake/cli/plugins/nativeapp/codegen/snowpark/models.py

```diff
@@ -28,7 +28,11 @@ class NativeAppExtensionFunction(_CallableBase):
     )
     packages: Optional[List[str]] = Field(
         title="List of packages (with optional version constraints) to be loaded for the function",
-        default={},
+        default=[],
+    )
+    raw_imports: Optional[List[Union[str, Tuple[str, str]]]] = Field(
```

Comment:
IMHO this should go to imports, not expose yet another field to represent the same data. This way we'll be consistent with Snowpark entities defined in the PDF. We can make the driver generate the correct list directly, as with the other fields.

Also, this change is unrelated to the DDL file generation change, I would have likely made 2 different PRs for these.